generated/nitclk/unit_tests/test_nitclk.py

```diff
@@ -0,0 +1,34 @@
+import _matchers
+import _mock_helper
+
+import nitclk
+
+from mock import patch
+
+SESSION_NUM_FOR_TEST = 42
```

Comment:
we're going to need to test all sorts of combinations... reason to stop accepting raw numbers. Ok for now, you could add a TODO.